app/views/vm/show.html.haml

```diff
@@ -11,6 +11,7 @@
     = render :partial => "vm_common/live_migrate"
   - elsif @evacuate
     = render :partial => "vm_common/evacuate"
+    = render :partial => 'layouts/angular/paging_div_buttons'
```

Comment:
All the other forms use the same angular buttons.

None of them use the partial this way, there is nothing special about evacuate, it should be using the same apporach as the rest.

Reply:
In explorer screens, the partial with the buttons is rendered because of [this line](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/vm_common.rb#L1235) of the code. For non explorer screens, there is not any code like this, `replace_right_cell` works only for explorer screens. We take care about non explorer screens differently. Also I think that we probably will need this render call also for other actions. So maybe the place for this call should be slightly different.